client/src/components/Rating.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+// Styling imports
+import { RatingWrapper, OverviewWrapper, OverviewText, CategoryWrapper, RatingContainer} from './styles/staticStyles.style'
+import { TextContainer, RatingText } from './styles/generalUse.style';
+import { Bar } from './styles/propStyles.style';
+import { FaStar } from "react-icons/fa";
+
+const Rating = (props) => {
+  const { cleanAvg, commAvg, accuracyAvg, valueAvg, locationAvg, checkinAvg } = props.overview;
+  const cleanPercent = Math.floor((Number(cleanAvg) / 5) * 100);
+  const commPercent = Math.floor((Number(commAvg) / 5) * 100);
+  const accuracyPercent = Math.floor((Number(accuracyAvg) / 5) * 100);
+  const valuePercent = Math.floor((Number(valueAvg) / 5) * 100);
+  const locationPercent = Math.floor((Number(locationAvg) / 5) * 100);
+  const checkinPercent = Math.floor((Number(checkinAvg) / 5) * 100);
+
+  return(
+    <RatingWrapper>
+
+      <OverviewWrapper><FaStar size='1em' color='#FF585D'/><OverviewText>{props.overview.avg} ({props.overview.reviewSize} reviews)</OverviewText></OverviewWrapper>
+
+      <CategoryWrapper>
+        <TextContainer>Cleaniness</TextContainer> 
+        <RatingContainer>
+          <Bar width='33%' color='#B3B3B3' height='0.25em' margin='5px'>
```

Comment:
seems like the Bar component is always called with the same props, so you can probably just hardcode those values inside the component to make this code cleaner. If it's only called with the same props most of the time I think you could still put them into the bar component as defaults.